.gitignore

```diff
@@ -26,9 +26,12 @@ yarn-error.log*
 
 # local env files
 .env.local
-.env.development.local
```

Comment:
don't remove .env.development.local from the gitignore. The reason it's there is so security things like usernames and passwords do not make their way into public repositories.